video-app/src/app/library/add/add.component.html

```diff
@@ -17,21 +29,25 @@
   </div>
   <div>
     <mat-button-toggle-group #platform="matButtonToggleGroup" value="youtube">
-      <mat-button-toggle id="youtube" name="youtube" value="youtube" aria-label="Text align center">
+      <mat-button-toggle value="youtube" aria-label="Text align center">
```

Comment:
jeżeli używasz aria-label to treść powinna być prawidłowo dobrana, tutaj jest to coś w stylu 'lorem ipsum'
albo usuń albo zmień na poprawną treść :P